boards/makerdiary/nrf52840_m2/nrf52840_m2.yaml

```diff
@@ -0,0 +1,25 @@
+identifier: nrf52840_m2
```

Comment:
@anangl think this is too common of a name and makerdiary should be in it or ok as-is?